classes/Kohana/Jam/Range.php

```diff
@@ -34,8 +34,8 @@ public static function sum(array $ranges, $format = NULL)
 
 		foreach ($ranges as $range)
 		{
-			$min += $range->min();
-			$max += $range->max();
+            $min += (int) $range->min();
+            $max += (int) $range->max();
```

Comment:
Yes. I've spot it last time. But this time had turned "ignore white space" comparison and missed it.

It's hard to use the same IDE settings for different projects with different styles. Will create another PR for adding .editorconfig to the project.